src/layouts/Titled/components/TOC.tsx

```diff
@@ -112,16 +114,19 @@ export const TOC: React.FC<{ data: IHeading[] }> = ({ data }) => {
           padding-left: ${p => p.theme.space.md};
         `}
       >
-        <SM
+        <div
           css={css`
             margin-left: -${p => p.theme.borderWidths.sm};
             text-transform: uppercase;
+            line-height: ${p => p.theme.lineHeights.sm};
             letter-spacing: 0.5px;
             color: ${p => getColor('grey', 600, p.theme)};
+            font-size: ${p => p.theme.fontSizes.xs};
+            font-weight: ${p => p.theme.fontWeights.semibold};
```

Comment:
Can you abstract out this common code repeated from above?

Reply:
Yes! I actually already had a component for this, but had missed it. Now sharing logic between the home page, desktop sidebar, mobile sidebar, and TOC sections.